src/parse/asp/builtins.go

```diff
@@ -198,8 +198,19 @@ func pkg(s *scope, args []pyObject) pyObject {
 	s.Assert(s.pkg.NumTargets() == 0, "package() must be called before any build targets are defined")
 	for k, v := range s.locals {
 		k = strings.ToUpper(k)
-		s.Assert(s.config.Get(k, nil) != nil, "error calling package(): %s is not a known config value", k)
-		s.config.IndexAssign(pyString(k), v)
+		configVal := s.config.Get(k, nil)
+		s.Assert(configVal != nil, "error calling package(): %s is not a known config value", k)
+
+		// Merge the config value together for plugins
+		if plugin := s.state.Config.GetPlugin(k); plugin != nil {
+			dict, ok := v.(pyDict)
+			s.Assert(ok, "error calling package(): %s is a plugin so that argument should be a dict", k)
```

Comment:
Why does this builtin function require knowledge of plugins, is it to display the type error via the assert?

Reply:
I just changed it to merge dictionaries together regardless of if they're plugin config

Reply:
Ya, I think it makes more sense too. Thanks.